spec/models/reaction_spec.rb

```diff
@@ -147,7 +150,9 @@
   context "when callbacks are called before destroy" do
     it "enqueues a ScoreCalcJob on article reaction destroy" do
       reaction = create(:reaction, reactable: article, user: user)
-      expect { reaction.destroy }.to have_enqueued_job(Articles::ScoreCalcJob).exactly(:once)
+      sidekiq_enqueued_jobs(queue: Articles::ScoreCalcJob) do
```

Comment:
`Articles::ScoreCalcJob` is not a sidekiq job
Also `sidekiq_enqueued_jobs` doesn't test anything, it's just a helper to get the jobs array, which is why test passes, because this test is not checking anything anymore

Reply:
So, should I keep this `expect` block?

Reply:
So, should I keep this `expect` block?

Reply:
Yes

Reply:
I'm receiving this error
```ruby
1) Reaction when callbacks are called before destroy enqueues a ScoreCalcJob on article reaction destroy
     Failure/Error: expect { reaction.destroy }.to have_enqueued_job(Articles::ScoreCalcJob).exactly(:once)
       expected to enqueue exactly 1 jobs, but enqueued 0
     # ./spec/models/reaction_spec.rb:153:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:95:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:95:in `block (2 levels) in <top (required)>'
```

I presume that it's because the worker isn't running. What do you think? @mstruve  @rhymes